keras_cv/models/backbones/resnet_v2/resnet_v2_backbone.py

```diff
@@ -341,19 +343,22 @@ def __init__(
         if stackwise_dilations is None:
             stackwise_dilations = [1] * num_stacks
 
-        stack_level_outputs = {}
+        pyramid_level_inputs = {}
         for stack_index in range(num_stacks):
+            num_blocks = stackwise_blocks[stack_index]
             x = apply_stack(
                 x,
                 filters=stackwise_filters[stack_index],
-                blocks=stackwise_blocks[stack_index],
+                blocks=num_blocks,
                 stride=stackwise_strides[stack_index],
                 dilations=stackwise_dilations[stack_index],
                 block_type=block_type,
                 first_shortcut=(block_type == "block" or stack_index > 0),
                 stack_index=stack_index,
             )
-            stack_level_outputs[stack_index + 2] = x
+            pyramid_level_inputs[
+                stack_index + 2
```

Comment:
While the literature uses "level 3", "level 4", etc --- I wonder if we should break that convention.  No strong opinion and lets not block on this comment - just a consideration

Reply:
No personal expertise but following the literature SGTM